src/redux/reducer/jobs.ts

```diff
@@ -0,0 +1,35 @@
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+import { http } from "../../util/config";
+import { DispatchType } from "../store/store";
+
+interface JobModal {
+    allJobs: JobModal[] | []
+}
```

Comment:
It is duplicated type name.
should be allJobs: JobModal[];

Reply:
i have already fixed